api/src/main/java/com/codeforcommunity/dto/site/GetSiteResponse.java

```diff
@@ -0,0 +1,66 @@
+package com.codeforcommunity.dto.site;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class GetSiteResponse {
+  private final Integer siteId;
+  private final Integer blockId;
+  private final BigDecimal lat;
+  private final BigDecimal lng;
+  private final String city;
+  private final String zip;
+  private final String address;
+  private final List<SiteEntry> entries;
+
+  public GetSiteResponse(
+      Integer siteId,
+      Integer blockId,
+      BigDecimal lat,
+      BigDecimal lng,
+      String city,
+      String zip,
+      String address,
+      List<SiteEntry> entries) {
+    this.siteId = siteId;
+    this.blockId = blockId;
+    this.lat = lat;
+    this.lng = lng;
+    this.city = city;
+    this.zip = zip;
+    this.address = address;
+    this.entries = entries;
+  }
+
+  public Integer getSiteId() {
+    return siteId;
+  }
+
+  public Integer getBlockId() {
+    return blockId;
+  }
+
+  public BigDecimal getLat() {
+    return lat;
+  }
+
+  public BigDecimal getLng() {
```

Comment:
⛏️ Maybe spelling these out as Latitude and Longitude might be more descriptive?

Reply:
This is consistent with the rest of the back end, but could be a ticket if you think it makes a big difference